src/odemis/gui/cont/tabs/fibsem_tab.py

```diff
@@ -349,7 +354,20 @@ def _stop_streams_subscriber(self):
     def _start_streams_subscriber(self):
         self.tab_data_model.streams.subscribe(self._on_acquired_streams)
 
-    def on_dbl_click(self, evt):
+    def on_sem_stage_movement(self, evt):
+        active_canvas = evt.GetEventObject()
+        logging.debug(f"mouse down event, canvas: {active_canvas}")
+
+        if not active_canvas._tab_data_model.tool.value and self.pm.current_posture.value != SEM_IMAGING:
+            logging.warning(("Moving the stage from the SEM stream is currently disabled. Please set the stage to the "
+                           "SEM posture to proceed. You can still use the FIB view for movement."
+                            "Warning"))
```

Comment:
Can you add a comment on why it's find to still propagate the event? Some other code will also detect this state and drop (silently) the move?